src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Infrastructure/WebApplicationBuilderExtensions.cs

```diff
@@ -45,6 +48,20 @@ public static void AddServices(this WebApplicationBuilder webAppBuilder)
         ConfigureDatastore(webAppBuilder, logger);
         ConfigureQueryHandler(webAppBuilder, logger);
 
+        webAppBuilder.Services.AddSingleton(
```

Comment:
> Why do you add this singleton for `DbHealthCheck` but I don't see one for `ApplicationHealthCheck`? Are you sure this is necessary?

It's because DbHealthCheck needs the ConnectionString and the provider